source/sessions/driver-sessions.rst

```diff
@@ -501,8 +501,6 @@ include a session ID in a ``KILLCURSORS`` command.
 How to Check Whether a Deployment Supports Sessions
 ===================================================
 
-Standalone servers do not support sessions.
-
```

Comment:
I think you should also replace "whether a replica set or sharded cluster deployment" with "whether a deployment" if we don't care about topologies anymore.